src/CommunityPatchLauncher/ViewModels/MainWindowModel.cs

```diff
@@ -146,6 +154,8 @@ public MainWindowModel(Window window) : base(window, true)
                 ReportIssueCommand = new OpenLinkCommand(wpfSettings.GetValue<string>("ReportIssueLink"));
                 OpenEditorCommand = new StartEditorCommand(settingManager);
                 OpenSettlersConfigurationCommand = new StartSettlersConfigCommand(settingManager);
+                OpenMPMapFolderCommand = new OpenFolderCommand(settingManager.GetValue<string>("GameFolder") + "/Map/User/");
+                OpenSavesFolderCommand = new OpenFolderCommand(Environment.GetFolderPath(Environment.SpecialFolder.MyDocuments) + "/TheSettlers4/Save/");
```

Comment:
Line seems a little bit long, maybe add a break so that it looks like this:

```
OpenSavesFolderCommand = new  penFolderCommand(
     Environment.GetFolderPath(
     Environment.SpecialFolder.MyDocuments
      ) + "/TheSettlers4/Save/"
);
```

Indent needs to be correct for sure